contracts/prize-strategy/PrizeStrategy.sol

```diff
@@ -852,12 +866,15 @@ contract PrizeStrategy is PrizeStrategyStorage,
   }
 
   modifier requireCanStartAward() {
+    require(_isPrizePeriodStarted(), "PrizeStrategy/prize-period-not-started");
```

Comment:
✂️ 

We already have isPrizePeriodOver()

Reply:
done